.rhcicd/clowdapp-backend.yaml

```diff
@@ -318,6 +324,12 @@ parameters:
 - name: SENTRY_ENABLED
   description: Enable Sentry (or not)
   value: "false"
+- name: QUARKUS_OPENTELEMETRY_ENABLED
+  value: "false"
+- name: QUARKUS_OPENTELEMETRY_ENDPOINT
+  value: "localhost:4317"
+- name: QUARKUS_DATASOURCE_JDBC_DRIVER
+  value: jdbc:postgresql://127.0.0.1:5432/notifications
```

Comment:
The param name mentions a driver while the value is a connection string. Which one do you want to configure here?

Reply:
This is what I want to ask you. I even believe that the Driver could always be the otel one, even if otel is disabled. It will then just be a thin wrapper around the real code.

Reply:
I agree that we can permanently use the otel driver and connection string (even if otel is disabled) as long as it looks stable on stage.